spec/const3.dd

```diff
@@ -468,8 +468,10 @@ $(LATEX
 )
 
 $(P In the graph above, any directed path is a legal implicit conversion. No
-other qualifier combinations than the ones shown is valid. The same information
-is shown below in tabular format:)
+other qualifier combinations than the ones shown is valid. If a directed path
+exists between two sets of qualifiers, the types thus qualified are called
```

Comment:
What is meant by "a directed path [...] between two sets of qualifiers"? I would just use what you have written in the glossary instead.

Reply:
There is a nice DAG on that page, sadly it does not have a number or a moniker. I'll look into it.